kmir/k-src/mir.md

```diff
@@ -515,6 +515,13 @@ or panics if the function-like or the block is missing:
         ...
        </k>
        <callStack> ListItem(FN_KEY) ... </callStack> [owise]
+
+  // Option Unwrap Call
+  rule <k> #executeTerminator(DEST_LOCAL:Local = Option :: < _TYPES > :: Unwrap :: .ExpressionPathList ( ARG , .OperandList ) -> ((NEXT:BBName _):BB)) 
+        => #executeStatement(DEST_LOCAL = #unwrap(ARG)) ~> #executeBasicBlock(FN_KEY, BBName2Int(NEXT)) ...
```

Comment:
Treating the assignment as a statement might not be a proper way of doing things. It could be confused that the origin was from an actual `Statement`, however it came from a `Terminator`. I personally think this is the best way to do things anyway.